includes/User.php

```diff
@@ -2416,11 +2417,183 @@ public function getIntOption( $oname, $defaultOverride=0 ) {
 		return intval( $val );
 	}
 
+
+	/**
+	 * Get a preference local to this wikia.
+	 *
+	 * @param string $preference the preference name
+	 * @param int $cityId the city id
+	 * @param string $sep the separator between the name and the city id
+	 * @return string
+	 */
+	public function getLocalPreference($pref, $cityId = null, $sep = "-") {
+		global $wgCityId;
+		if (!isset($cityId)) {
+			$cityId = $wgCityId;
+		}
```

Comment:
@drsnyder I think I'm missing something ... shouldn't there be more code here?  Something to actually get the preference value?